virttest/qemu_vm.py

```diff
@@ -3710,6 +3712,11 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
 
             logging.info("Migrating to %s", uri)
             self.monitor.migrate(uri)
+
+            if mig_inner_cmd:
+                for func in mig_inner_cmd:
+                    func()
+
```

Comment:
From the name 'mig_inner_cmd', I suppose it's a set of commands
but It's a set of functions, right？
Maybe we need an appropriate and precise name for this parameter.